app/views/items/collection_ui.html.erb

```diff
@@ -1,3 +1,5 @@
+<%= javascript_include_tag 'show' %>
```

Comment:
So I did move the file into the `app/assets/javascripts/modules` folder and the JS is being included correctly, but because the elements I am binding my `.click` events to aren't in the DOM until the user opens the Edit Collections modal, the binding code is never being triggered.  Not sure how to handle that.

Reply:
never mind...figured it out --- now using `.on('click'...)` binding which covers binding DOM elements that are added later).